apps/prosody/templates/services/app/deploy.yml.j2

```diff
@@ -0,0 +1,74 @@
+apiVersion: v1
+kind: Deployment
+metadata:
+  labels:
+    app: prosody
+    service: app
+    version: "{{ prosody_image_tag }}"
+    deployment_stamp: "{{ deployment_stamp }}"
+  name: "prosody-app"
+  namespace: "{{ namespace_name }}"
+spec:
+  replicas: {{ prosody_replicas }}
+  selector:
+    matchLabels:
+      app: prosody
+      service: app
+      deployment: "prosody-app"
+  template:
+    metadata:
+      labels:
+        app: prosody
+        service: app
+        deployment: "prosody-app"
+    spec:
+      # Prefer running pods on different nodes for redundancy
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 100
+              podAffinityTerm:
+                labelSelector:
+                  matchExpressions:
+                    - key: deployment
+                      operator: In
+                      values:
+                        - "prosody-app"
+                topologyKey: kubernetes.io/hostname
+{% set image_pull_secret_name = prosody_image_pull_secret_name | default(none) or default_image_pull_secret_name %}
+{% if image_pull_secret_name is not none %}
+      imagePullSecrets:
+        - name: "{{ image_pull_secret_name }}"
+{% endif %}
+      containers:
```

Comment:
No `securityContext` is defined for this deployment, is it intentional?

Reply:
Yes, I want to use the `prosody` user defined in the image.